Source/Public/Convert-JsonToBicep.ps1

```diff
@@ -0,0 +1,41 @@
+function Convert-JsonToBicep {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory, 
+            ParameterSetName = 'String')]
+        [ValidateNotNullOrEmpty()]
+        [ValidateScript({$_ | Convertfrom-Json})]
+        [string]$String
+    )
+
+    begin {        
+        $FileResolver = [Bicep.Core.FileSystem.FileResolver]::new()
+        $ResourceProvider = [Bicep.Core.TypeSystem.Az.AzResourceTypeProvider]::new()
+    }
+
+    process {
+        $jsonObject = ConvertFrom-Json -InputObject $String -AsHashtable -Depth 100
+        $variables = [ordered]@{}
+        $templateBase = [ordered]@{
+            '$schema'        = 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+            'contentVersion' = '1.0.0.0'
+        }
+        $variables['temp'] = $jsonObject                                  
+        $templateBase['variables'] = $variables
+        $tempTemplate = ConvertTo-Json -InputObject $templateBase -Depth 100
+        Out-File -InputObject $tempTemplate -FilePath "$($env:TEMP)\tempfile.json"
```

Comment:
If we need to create a tempfile, we should clean up after the process is done. Please add Remove-Item after the file has been used.